File: pinyougou-web-shop/src/main/webapp/js/service/roomService.js
//服务层
app.service('roomService',function($http){
	    	
	//查询
	this.findAll=function(){
		return $http.get('../room/findAll.do');		
	}
	//分页
	this.findPage=function(page,rows){
		return $http.get('../room/findPage.do?page='+page+'&rows='+rows);
	}

	//增加 
	this.add=function(entity){
		return  $http.post('../room/add.do',entity );
	}

	//删除
    this.dele = function(ids){
        return $http.get("../room/delete.do?ids="+ids);
    }
});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/room/RoomService.java
package cn.itcast.core.service.room;

import cn.itcast.core.pojo.room.Room;

import java.util.List;

public interface RoomService {

    //新增会议室
    void add(Room room);

    //查询所有
    List<Room> findAll();

    //删除资源
    public void delete(Long[] ids);
}
